tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php

```diff
@@ -952,6 +952,135 @@ public function provideMessyWhitespacesCases()
         );
     }
 
+    /**
+     * @param array       $config
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideSwitchCases
+     */
+    public function testInSwitchStatement(array $config, $expected, $input = null)
+    {
+        $this->fixer->configure(array('tokens' => $config));
+        $this->doTest($expected, $input);
+    }
+
+    public function provideSwitchCases()
+    {
+        return array(
+            array(
+                array(
+                    'break',
+                    'continue',
+                    'extra',
+                    'return',
+                    'throw',
+                ),
+                '<?php
+                    /** a  */
+                    switch ($a) {
+                        case 1:
+                            break;
+                        case 2:
+                            continue;
+                        case 3:
+                            return 1;
+                        case 4:
+                            throw $e;
+                        case 5:
+                            throw new \Exception();
+                        case Token::STRING_TYPE:
+                            echo 123;
+    
+                            return new ConstantNode($token->getValue());
+                        case 7:
+                            return new ConstantNode($token->getValue());
+                        case 8:
+                            return 8;
+                        default:
+                            echo 1;
+                    }',
+                '<?php
```

Comment:
**@stof I've tried to recreate the issue you reported (https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2697) here, however it does not fail for me. Can you confirm you have both `throws` and `return` configured on the fixer?**